restord/ImageSessionServer.m

```diff
@@ -435,6 +435,16 @@ - (int)runPostScriptWithMountURL:(NSURL *)mountURL error:(NSError **)error {
 
   NSError *err;
   NSFileManager *fm = [NSFileManager defaultManager];
+
+  // Paranoid about privleged escalation. If the dir already exists, really remove it.
+  BOOL isDir = NO;
+  if ([fm fileExistsAtPath:directoryURL.path isDirectory:&isDir] && isDir) {
```

Comment:
I was going to say that it might be better to just try removeItemAtURL without checking first, and ignore the error if it is of the "no such file or directory" variety, but it seems like it's really difficult to extract this information out of the NSError, so I guess never mind.

Reply:
Yeah, I went down that road earlier ;)